probe-rs/src/config/mod.rs

```diff
@@ -1,19 +1,45 @@
+#![warn(missing_docs)]
+
+//! Target specific configuration
```

Comment:
```suggestion
//! # Target specific configuration
```

Reply:
I prefer it if it's not formatted as a title, so this is on purpose.

Reply:
Uhm I think this looks totally weird if it's just formatted like normal text. But I guess it doesn't matter too much :)